### PR TITLE
Add information about wallabag:user:show

### PR DESCRIPTION
**Pull request by j0k3r**
See https://github.com/wallabag/wallabag/pull/3179

Ping @wallabag/translators 🙂 

- [ ] 🇮🇹 
- [ ] 🇩🇪

### CODE REVIEW

File: fr/admin/console_commands.md [hunk 1]
```diff
@@ -141,3 +141,18 @@ wallabag:tag:all <username>
...
+Cette commande vous permet d'afficher les détails d'un utilisateur.
+
+Utilisation:
+
+```
+wallabag:user:show <username>
+```
+
+Arguments:
+ - username: Utilisateur sur lequel travailler.
```
Comment (tcitworld) [inline fr/admin/console_commands.md]: 🤔

Comment (j0k3r) [inline fr/admin/console_commands.md]: Utilisateur pour lequel vous voulez des informations?

Comment (tcitworld) [inline fr/admin/console_commands.md]: Utilisateur à afficher ?

File: en/admin/console_commands.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Custom to wallabag:
  - `wallabag:import:redis-worker`: Useful if you use Redis.
  - `wallabag:install`: (re)Install wallabag
  - `wallabag:tag:all`: Tag all entries for an user using his/her tagging rules.
+ - `wallabag:user:show`: Shows the details for an user.
```
Comment (nicosomb) [inline en/admin/console_commands.md]: for `a` user.

File: en/admin/console_commands.md [hunk 3]
```diff
@@ -137,3 +138,18 @@ wallabag:tag:all <username>
 
 Arguments:
  - username: User to tag entries for.
+
+
+wallabag:user:show
+------------------
+
+This command shows the details for an user.
```
Comment (nicosomb) [inline en/admin/console_commands.md]: same as above.

File: en/admin/console_commands.md [hunk 4]
```diff
@@ -137,3 +138,18 @@ wallabag:tag:all <username>
 
 Arguments:
  - username: User to tag entries for.
+
```
Comment (nicosomb) [inline en/admin/console_commands.md]: extra line?

File: en/admin/console_commands.md (hunk 2 above)
Comment (j0k3r) [inline en/admin/console_commands.md]: Is that an exception? A `a` before a word starting with a vowel shouldn't be converted to `an`?

File: en/admin/console_commands.md (hunk 4 above)
Comment (j0k3r) [inline en/admin/console_commands.md]: Yeah, I put 2 new lines between each command.
Should I remove them?

File: en/admin/console_commands.md (hunk 2 above)
Comment (tcitworld) [inline en/admin/console_commands.md]: English teacher's son here. Definitely should.

File: en/admin/console_commands.md (hunk 4 above)
Comment (nicosomb) [inline en/admin/console_commands.md]: not necessary.

**Comment by matteocoder** [comment]
Good! Just change "italiana" to "italiano" though :)

**Pull request merged**